Documentation/op-guide/failures.md

```diff
@@ -6,7 +6,7 @@ In this section, we catalog kinds of failures and discuss how etcd is designed t
 
 ## Minor followers failure
 
-When fewer than half of the followers fail, the etcd cluster can still accept requests and make progress without any major disruption. For example, two follower failures will not affect a five member etcd cluster’s operation. However, clients will lose connectivity to the failed members. Client libraries should hide these interruptions from users for read requests by automatically reconnecting to other members. Operators should expect the system load on the other members to increase due to the reconnections.
```

Comment:
I would say, instead of saying followers, we should say members and the title should be changed as well to Minor members failure.

Reply:
I'm afraid that "minor members" can include the leader. As described in the next section, perhaps we should distinguish the necessity of leader election.

Reply:
When etcd has 6 members (1 leader, and 5 followers), it can only tolerate 2 followers failures. But the half of 5 is 2.5. So it might be a little bit confusing.

Reply:
Actually 2 <= 2.5 holds, so it doesn't matter. IMHO, the inconsistency with the "2/5 failure" example is more confusing because 2 followers are exactly a half of followers. How is describing more explicit, something like:

When fewer than (or equal to, if the cluster has odd members) half of the followers fail...

Reply:
that would be better.

Reply:
Excuse me, but the ordering of comments looks confused.

@xiang90 
Which idea would be better you mean?

@raoofm 
Yes I can understand your opinion that it doesn't matter for tolerance the node is the leader or a follower. But the document aims to classify failures. From this point of view, leader election is notable because it requires a period of read-only.